app/models/govbox_set.rb

```diff
@@ -138,6 +140,7 @@ def format_graphics_best_bet
 
   def format_jobs
     return unless @affiliate.jobs_enabled?
+    return I18n.t(:no_job_results) if @jobs.blank?
```

Comment:
I don't think there is not need to return this. The frontend already has all of them and it can display it if jobs is empty.

Reply:
As per the requirement Yogesh wanted me to return something to know when there are no jobs. The reason I am validating jobs above as string is because I am returning the string here which is failing without validation.

Reply:
Do you think returning an empty array would be good enough to know there are no jobs? what about `nil`?

Reply:
If an empty array or `nil` are returned they'll be removed from the JSON due to the `#compact_blank` in `#as_json`.

Reply:
If we go this route (using `I18n.t()` we'll need to update the types/props in the relevant component(s).